net.sourceforge.fuse_emulator.Fuse.appdata.xml

```diff
@@ -0,0 +1,105 @@
+          <li>Improved Kempston mouse handling</li>
+          <li>Several UI fixes and improvements</li>
+        </ul>
+      </description>
+    </release>
+    <release date="2018-08-08" version="1.5.6">
+      <description>
+        <ul>
+          <li>Z80 flags register is now correct after SCF and CCF</li>
+          <li>Accelerate Alkatraz, Search and Dinaload loaders</li>
+          <li>Bug fixes</li>
+        </ul>
+      </description>
+    </release>
+    <release date="2018-07-01" version="1.5.5">
+      <description>
+        <ul>
+          <li>Improve sound accuracy on Unix 64 bits systems</li>
+          <li>Bug fixes</li>
+        </ul>
+      </description>
+    </release>
+    <release date="2018-06-06" version="1.5.4">
+      <description>
+        <ul>
+          <li>Bug fixes</li>
+        </ul>
+      </description>
+    </release>
+  </releases>
+  <description>
+    <p>
+      Fuse is an emulator of the Sinclair ZX Spectrum home
+      computer. It supports all of the original models and some
+      clones, with quite faithful emulation of the display and
+      sound.
+    </p>
+    <p>
+      Fuse can load most formats of snapshots, tapes and floppy disks
+      and it can emulate many peripherals such as joysticks, printers,
+      storage devices and network interfaces, among others.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Fuse running the 128 BASIC editor</caption>
+      <image>
+        https://raw.githubusercontent.com/flathub/net.sf.fuse_emulator/6b92a936f359da8e834079bc132e82ea3f8e31b3/screenshots/fuse1.png
+      </image>
+    </screenshot>
+    <screenshot>
+      <caption>Fuse loading The Lords of Midnight</caption>
+      <image>
+        https://raw.githubusercontent.com/flathub/net.sf.fuse_emulator/6b92a936f359da8e834079bc132e82ea3f8e31b3/screenshots/fuse2.png
+      </image>
+    </screenshot>
+    <screenshot>
+      <caption>Fuse running Head Over Heels</caption>
+      <image>
+        https://raw.githubusercontent.com/flathub/net.sf.fuse_emulator/6b92a936f359da8e834079bc132e82ea3f8e31b3/screenshots/fuse3.png
+      </image>
+    </screenshot>
+    <screenshot>
+      <caption>The Fuse debugger window</caption>
+      <image>
+        https://raw.githubusercontent.com/flathub/net.sf.fuse_emulator/6b92a936f359da8e834079bc132e82ea3f8e31b3/screenshots/fuse4.png
+      </image>
+    </screenshot>
+  </screenshots>
+  <developer id="net.sourceforge.fuse-emulator"><name>The Fuse development team</name></developer>
+  <update_contact>fuse-emulator-devel_AT_lists.sourceforge.net</update_contact>
+  <url type="homepage">https://fuse-emulator.sourceforge.net/</url>
+  <url type="bugtracker">https://sourceforge.net/p/fuse-emulator/bugs/</url>
+  <content_rating type="oars-1.1"/>
+  <launchable type="desktop-id">net.sourceforge.fuse_emulator.Fuse.desktop</launchable>
+  <launchable type="desktop-id">net.sourceforge.fuse_emulator.Fuse.SDL.desktop</launchable>
```

Comment:
```suggestion
```

Add only the main desktop file matching the ID.

Reply:
Sure, I was told [some months ago](https://github.com/flathub/net.sf.fuse_emulator/pull/9#issuecomment-2002265075) to add two `<launchable>` tags if I had two `.desktop` files, but I see [in the spec](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-launchable) that I should try to avoid that.

I'll keep only the main one.